WalletWasabi.Fluent/Views/Wallets/Send/TransactionPreviewView.axaml

```diff
@@ -67,73 +67,78 @@
 
-                          <Setter Property="BorderBrush" Value="#2AFFFFFF" />
-                          <Setter Property="BorderThickness" Value="1" />
-                          <Setter Property="Background" Value="{StaticResource OptimizePrivacyOptionSelectedColor}" />
-                          <Setter Property="BoxShadow" Value="{StaticResource OptimizePrivacyOptionBoxShadow2}" />
+                            <Setter Property="BorderBrush" Value="#2AFFFFFF" />
+                            <Setter Property="BorderThickness" Value="1" />
+                            <Setter Property="Background" Value="{StaticResource OptimizePrivacyOptionSelectedColor}" />
+                            <Setter Property="BoxShadow" Value="{StaticResource OptimizePrivacyOptionBoxShadow2}" />
 
-                          <Setter Property="Opacity" Value="0" />
-                        </Style>
+                            <Setter Property="Opacity" Value="0" />
+                          </Style>
 
-                        <Style Selector="ListBoxItem:pointerover Border#PART_DecorationBorderHover">
-                          <Setter Property="Opacity" Value="1" />
-                        </Style>
-                      </ListBox.Styles>
-                      <ListBox.ItemsPanel>
-                        <ItemsPanelTemplate>
-                          <WrapPanel Orientation="Horizontal" />
-                        </ItemsPanelTemplate>
-                      </ListBox.ItemsPanel>
-                      <ListBox.DataTemplates>
-                        <DataTemplate DataType="vm:ChangeAvoidanceSuggestionViewModel">
-                          <c:SuggestionItem>
-                            <send:ChangeAvoidanceSuggestionControl />
-                          </c:SuggestionItem>
-                        </DataTemplate>
-                      </ListBox.DataTemplates>
-                    </ListBox>
+                          <Style Selector="ListBoxItem:pointerover Border#PART_DecorationBorderHover">
+                            <Setter Property="Opacity" Value="1" />
+                          </Style>
+                        </ListBox.Styles>
+                        <ListBox.ItemsPanel>
+                          <ItemsPanelTemplate>
+                            <WrapPanel Orientation="Horizontal" />
+                          </ItemsPanelTemplate>
+                        </ListBox.ItemsPanel>
+                        <ListBox.DataTemplates>
+                          <DataTemplate DataType="vm:ChangeAvoidanceSuggestionViewModel">
+                            <c:SuggestionItem>
+                              <send:ChangeAvoidanceSuggestionControl />
+                            </c:SuggestionItem>
+                          </DataTemplate>
+                        </ListBox.DataTemplates>
+                      </ListBox>
+
+                      <c:ProgressRing Height="100" Width="100" IsIndeterminate="True" IsVisible="{Binding IsBusy}" Margin="0 20 0 0" />
```

Comment:
Use the same spinner that `LoadingSuggestionView.axaml` uses, with a tiny margin:

```xml
<Viewbox Height="50" VerticalAlignment="Center" Margin="25">
  <c:ProgressRing Height="100" Width="100" IsIndeterminate="True" />
</Viewbox>
```